src/Data/List/Relation/Unary/All.agda

```diff
@@ -60,7 +60,7 @@ data _[_]=_ {A : Set a} {P : Pred A p} :
 -- A list is empty if having an element is impossible.
 
 Null : Pred (List A) _
-Null = All (λ _ → ⊥)
+Null {ℓ} = All (λ _ → ⊥ {ℓ})
```

Comment:
Again what does this add?

Reply:
I think this was needed because it was lifted somewhere else that needed this. I'll investigate and revert if it doesn't add anything.

Reply:
Yeah, that was just pointless.